src/renderer/js/modal/modalCreditIntro/view.jsx

```diff
@@ -4,55 +4,42 @@ import { CreditAmount, CurrencySymbol } from "component/common";
 import Link from "component/link/index";
 
 const ModalCreditIntro = props => {
-  const { closeModal, totalRewardValue, currentBalance, verifyAccount } = props;
+  const { closeModal, totalRewardValue, currentBalance, addBalance } = props;
   const totalRewardRounded = Math.round(totalRewardValue / 10) * 10;
 
   return (
     <Modal type="custom" isOpen={true} contentLabel="Welcome to LBRY">
       <section>
-        <h3 className="modal__header">{__("Blockchain 101")}</h3>
+        <h3 className="modal__header">
+          {__("Computer Wizard Needs Tokens Badly")}
+        </h3>
         <p>
-          LBRY is controlled and powered by a blockchain asset called{" "}
+          Some actions require{" "}
```

Comment:
as far as the i18n goes, why is some text not inside __(...)?

Reply:
Because our i18n system doesn't allow me to embed other elements. It's debatable whether a translation fragment is better or worse than not translating at all.

Reply:
Ah makes sense, yeah I think translation fragments aren't great.